diff --git a/src/terminal-factory-service.xml b/src/terminal-factory-service.xml
index 53b0acf..afe168e 100644
--- a/src/terminal-factory-service.xml
+++ b/src/terminal-factory-service.xml
@@ -1,5 +1,5 @@
-<node name="/org/gnome/Terminal">
-  <interface name="org.gnome.Terminal.Factory">
+<node name="/org/flatcap/Terminal">
+  <interface name="org.flatcap.Terminal.Factory">
     <method name="HandleArguments">
       <arg type="ay" name="working_directory" direction="in" />
       <arg type="ay" name="display_name" direction="in" />
diff --git a/src/terminal.c b/src/terminal.c
index 17de783..5f09c1e 100644
--- a/src/terminal.c
+++ b/src/terminal.c
@@ -48,9 +48,9 @@
 #include "terminal-options.h"
 #include "terminal-util.h"
 
-#define TERMINAL_FACTORY_SERVICE_NAME_PREFIX  "org.gnome.Terminal.Display"
-#define TERMINAL_FACTORY_SERVICE_PATH         "/org/gnome/Terminal/Factory"
-#define TERMINAL_FACTORY_INTERFACE_NAME       "org.gnome.Terminal.Factory"
+#define TERMINAL_FACTORY_SERVICE_NAME_PREFIX  "org.flatcap.Terminal.Display"
+#define TERMINAL_FACTORY_SERVICE_PATH         "/org/flatcap/Terminal/Factory"
+#define TERMINAL_FACTORY_INTERFACE_NAME       "org.flatcap.Terminal.Factory"
 
 #define TERMINAL_TYPE_FACTORY             (terminal_factory_get_type ())
 #define TERMINAL_FACTORY(object)          (G_TYPE_CHECK_INSTANCE_CAST ((object), TERMINAL_TYPE_FACTORY, TerminalFactory))
@@ -396,7 +396,7 @@ main (int argc, char **argv)
                                          factory_name,
                                          TERMINAL_FACTORY_SERVICE_PATH,
                                          TERMINAL_FACTORY_INTERFACE_NAME);
-      retval = org_gnome_Terminal_Factory_handle_arguments (proxy,
+      retval = org_flatcap_Terminal_Factory_handle_arguments (proxy,
                                                             working_directory_array,
                                                             display_name_array,
                                                             startup_id_array,
